pkg/kubelet/config/config.go

```diff
@@ -254,16 +254,16 @@ func (s *podStorage) merge(source string, change interface{}) (adds, updates, de
 	switch update.Op {
 	case kubetypes.ADD, kubetypes.UPDATE, kubetypes.DELETE:
 		if update.Op == kubetypes.ADD {
-			klog.V(4).Infof("Adding new pods from source %s : %v", source, update.Pods)
+			klog.V(4).InfoS("Added new pods from source", "source", source, "pods", update.Pods)
```

Comment:
Could you use `format.Pods` from https://github.com/kubernetes/kubernetes/pull/99799